src/BuildIt.Forms/BuildIt.Forms.Controls/Platforms/Android/CameraPreviewControlRenderer.cs

```diff
@@ -182,13 +182,18 @@ protected override void OnLayout(bool changed, int l, int t, int r, int b)
             view.Layout(0, 0, r - l, b - t);
         }
 
-        private static CameraFacing ToCameraFacing(CameraPreviewControl.CameraPreference cameraPreference)
+        private static CameraFacing ToCameraFacing(CameraPreviewControl.CameraFacing cameraPreference)
         {
-            return cameraPreference == CameraPreviewControl.CameraPreference.Back
+            return cameraPreference == CameraPreviewControl.CameraFacing.Back
                 ? CameraFacing.Back
                 : CameraFacing.Front;
         }
 
+        private static CameraPreviewControl.CameraFacing ToCameraFacing(CameraFacing cameraFacing)
```

Comment:
Is this the same as the previous method?

Reply:
no, the enum in Android and BuildIt is the same name so there are separate methods to go from one to the other